### PR TITLE
Added mem plugin

### PR DESCRIPTION
**Pull request by jeremija**
Hi there, 

I implemented a plugin for displaying RAM usage. I'm not 100% sure everything is right, but it seems to work. I'd be happy to hear your comments.

Here is a sample `j4status.conf`:

```
[Plugins]
Output=i3bar
Input=mem

[Override mem]
Label=MEM

[Memory]
Frequency=5
```

Thanks.

### CODE REVIEW

File: mem/src/mem.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    struct sysinfo meminfo;
+    sysinfo(&meminfo);
+
+    unsigned long used_ram = meminfo.totalram - meminfo.freeram -
+        meminfo.sharedram - meminfo.bufferram;
+    double mem_percent = 100.0 * used_ram / meminfo.totalram;
+
+    j4status_section_set_state(context->section,
+        mem_percent < 50 ? J4STATUS_STATE_NO_STATE :
```
Comment (sardemff7) [inline mem/src/mem.c]: Why `NO_STATE` and not `GOOD`?

**Comment by sardemff7** [comment]
Just one comment inline, otherwise looks pretty good.
Will push once fixed/answered.

Be prepared to add some configuration as people might request them. :-)
Some ideas: to show real/virtual RAM, as a percentage or as a full value, to configure the thresholds (people with less RAM and a quite heavy workflow might be at 70% minimum).

**Comment by jeremija** [comment]
@sardemff7 Thanks! I've just pushed a commit with configurable thresholds: `GoodThreshold` and `BadThreshold`. I put `NO_STATE` because I was trying to mimic the behavior of the `cpu` plugin.

**Comment by sardemff7** [comment]
Ideally, commit messages should be in the present, not past.
I pushed your plugin.

I will add you as a contributor to the repository, so you can push fixes and handle bug reports for your plugin. Please, if possible, do not use the “Merge pull request” feature of GitHub and instead merge it manually, fast-forward, to avoid the annoying merge commit when possible (and ask for a rebase on pull requests to have the most fast-forward possible).

Thanks!

**Comment by jeremija** [comment]
Noted! Thanks for accepting the pull request and adding me to the repository!

**Pull request merged**